src/js/letters/utils/helpers.js

```diff
@@ -58,21 +37,169 @@ export const letterContent = {
+      dependent: <div>You <strong>are not</strong> receiving Dependency and Indemnity Compensation</div>
+    }
+  },
+  hasSurvivorsPensionAward: {
+    'true': {
+      veteran: undefined,
+      dependent: <div>You <strong>are</strong> receiving Survivors Pension></div>
+    },
+    'false': {
+      veteran: undefined,
+      dependent: <div>You <strong>are not</strong> receiving Survivors Pension></div>
+    }
+  },
+  hasAdaptedHousing: {
+    'true': {
+      veteran: <div>You <strong>have</strong> been found entitled to a Specially Adapted Housing (SAH) and/or Special Home Adaptation (SHA) grant</div>,
+      dependent: undefined
+    },
+    'false': {
+      veteran: undefined,
+      dependent: undefined
+    }
+  },
+  hasChapter35Eligibility: {
+    'true': {
+      veteran: <div>You are considered to be totally and permanently disabled solely due to your service-connected disabilities</div>,
+      dependent: <div>You <strong>are</strong> eligible for Dependents' Educational Assistance (Chapter 35).</div>
+    },
+    'false': {
+      // This doesn't look right: getting clarification in
+      // https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3685
+      veteran: <div>Information on your totally and permanently disabled status which resulted from service-connected disabilities</div>,
+      dependent: <div>You <strong>are not</strong> eligible for Dependents' Educational Assistance (Chapter 35).</div>
+    }
+  },
+  hasDeathResultOfDisability: {
+    'true': {
+      veteran: undefined,
+      dependent: <div>The Veteran died as a result of a service-connected disability</div>
+    },
+    'false': {
+      veteran: undefined,
+      dependent: <div>The Veteran did not die as a result of a service-connected disability</div>
+    }
+  },
+  hasIndividualUnemployabilityGranted: {
+    'true': {
+      veteran: <div>You are being paid at the 100 percent rate because you are unemployable due to your service-connected disabilities</div>,
+      dependent: undefined
+    },
+    'false': {
+      veteran: undefined,
+      dependent: undefined
+    }
+  },
+  hasSpecialMonthlyCompensation: {
+    'true': {
+      veteran: <div>You are service-connected for loss of or loss of use of a limb, or you are totally blind in or missing at least one eye</div>,
+      dependent: undefined
+    },
+    'false': {
+      veteran: undefined,
+      dependent: undefined
+    }
+  }
+};
+
+export function getBenefitOptionText(option, value, isVeteran) {
+  const personType = isVeteran ? 'veteran' : 'dependent';
+  let valueString;
+  if (value === false) {
+    valueString = 'false';
+  } else if (value === true) {
+    valueString = 'true';
+  } else {
+    valueString = value;
+  }
+
+  if (!['awardEffectiveDate', 'monthlyAwardAmount', 'serviceConnectedPercentage'].includes(option)) {
+    return benefitOptionText[option][valueString][personType];
+  }
+  switch (option) {
+    case 'awardEffectiveDate':
```

Comment:
Not sure we have to worry about `awardEffectiveDate` (see comment below), let's confirm with @rroueche